### PR TITLE
schedule cooldown_times  and host_numbers  field can not support more than 16 entries

### PR DESCRIPTION
**Pull request by sulixu**
varchar(32) to varchar(2048)

### CODE REVIEW

File: tools/mysql/schema-update-2.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+-- !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+-- ALWAYS BACKUP YOUR DATA BEFORE EXECUTING THIS SCRIPT
+-- !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+# This script upgrade DB schema from version 1 to version 2
+
+ALTER TABLE schedules MODIFY COLUMN cooldown_times VARCHAR(2048);
```
Comment (lilida) [inline tools/mysql/schema-update-2.sql]: I think you should also update deploy.sql

File: tools/mysql/schema-update-2.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+ALTER TABLE schedules MODIFY COLUMN host_numbers VARCHAR(2048);
+
+-- make sure to update the schema version to current+1
+UPDATE schema_versions SET version=2;
```
Comment (lilida) [inline tools/mysql/schema-update-2.sql]: Also need to update the version in deploy.sql

File: tools/mysql/schema-update-2.sql (hunk 1 above)
Comment (sulixu) [inline tools/mysql/schema-update-2.sql]: thanks, done

File: deploy-service/common/src/main/resources/sql/deploy.sql [hunk 3]
```diff
@@ -269,8 +269,8 @@ CREATE INDEX tags_target_type_idx ON tags(target_type, created_date);
 CREATE TABLE IF NOT EXISTS schedules (
     id                  VARCHAR(22)     NOT NULL,
     total_sessions      INT             NOT NULL DEFAULT 0,
-    cooldown_times      VARCHAR(32)     NOT NULL,
-    host_numbers        VARCHAR(32)     NOT NULL,
+    cooldown_times      VARCHAR(2048)     NOT NULL,
+    host_numbers        VARCHAR(2048)     NOT NULL,
```
Comment (lilida) [inline deploy-service/common/src/main/resources/sql/deploy.sql]: Also update the last line of deploy.sql 
INSERT INTO schema_versions (version) VALUES (1); <<--- Change this to 2

**Pull request closed without merging**